packages/realm/src/platform/react-native/device-info.ts

```diff
@@ -19,6 +19,9 @@
 import { Platform } from "react-native";
 
 import { version } from "realm/package.json";
+// eslint-disable-next-line @typescript-eslint/ban-ts-comment
+// @ts-ignore
```

Comment:
I've got the same question as the previous one 🙂

Reply:
Same answer